src/main/webapp/navbar.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function rebuildNavBar(currentUser) {
+  const navBar = document.getElementById("nav-bar");
+  navBar.textContent = "";
+
+  const siteTitle = document.createElement("div");
+  siteTitle.setAttribute("id", "site-title");
+  siteTitle.textContent = "Givr";
+  navBar.appendChild(siteTitle);
+
+  const navLinksArea = document.createElement("div");
+  navLinksArea.setAttribute("id", "nav-links");
+
+  const getHelpLink = document.createElement("a");
+  getHelpLink.setAttribute("href", "index.html");
+  getHelpLink.textContent = "Get Help";
+  navLinksArea.appendChild(getHelpLink);
+
+  const eventsLink = document.createElement("a");
+  eventsLink.setAttribute("href", "events.html");
+  eventsLink.textContent = "Events";
+  navLinksArea.appendChild(eventsLink);
+
+  const registerOrganizationLink = document.createElement("a");
+  registerOrganizationLink.setAttribute("href", "register_organization.html");
+  registerOrganizationLink.textContent = "Register Organization";
+  navLinksArea.appendChild(registerOrganizationLink);
```

Comment:
x = new URL(window.location.href)
x.pathname

will give you the current path (the thing after the domain). How can you use that to do the "selected" state you were talking about (not in this PR)